_data/sidebars/home_sidebar.yml

```diff
@@ -83,9 +83,31 @@ entries:
       url: /what-are-things.html
       output: web
 
-    - title: What Are The Admin Things
-      url: /what-are-admin-things.html
-      output: web
+      subfolders:
+      - title: What Are The Admin Things
```

Comment:
I'm not a fan of the "Admin Things" grouping.  I think this is too broad.  I don't understand what belongs here an what doesn't.  How about "Users and Roles"?